plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -481,11 +481,14 @@ export class SessionRecordingIngester {
         // eachBatchWithContext, then commits offsets for the batch.
         // the batch consumer reads from the session replay kafka cluster
         const replayClusterConnectionConfig = createRdConnectionConfigFromEnvVars(this.config)
+
+        const autoCommit = this.config.SESSION_RECORDING_USE_OFFSET_STORE ? true : false
```

Comment:
Ternary expression on a boolean, very thorough :)

Reply:
YOU NEVER KNOW :p

Reply:
actually just noticed this won't work...

Reply:
??

Reply:
Updated - I realised that that consumer code has a block for doing the offset store update (which would be super bad!). Added a param to bypass it so that the replay one is in charge

Reply:
Ah, sorry, I thought you were on your own consumer entirely. LGTM